firmware/gen_signature.sh

```diff
@@ -10,12 +10,6 @@ echo "Generating signature for ${SHORT_BOARDNAME}"
 
 # generate a unique signature
 date=`TZ=Europe/London date +"%Y.%m.%d"`
-echo "// Generated by gen_signature.sh" > ${SIGNATURE_FILE_NAME}
-echo "#define SIGNATURE_DATE $date" >> ${SIGNATURE_FILE_NAME}
-echo "#define SIGNATURE_BOARD ${SHORT_BOARDNAME}" >> ${SIGNATURE_FILE_NAME}
-
-echo "// SIGNATURE_HASH is a built-in variable generated by ConfigDefinition.jar" >> ${SIGNATURE_FILE_NAME}
```

Comment:
I am attached to this line to be honest

my problem is I do not remember all the gears in our monster of mechanism. I need things self-documenting

What's the big picture here, what is driving the change?

I am happy to throw away _date and _board if unused

Reply:
Neither `_date` nor `_board` exist anywhere in the firmware other than in this script, and in the generated headers.